plugins/node.d/http_loadtime.in

```diff
@@ -11,20 +11,26 @@ http_loadtime - Plugin to graph the HTTP response times of specific pages
 The following environment variables are used by this plugin
 
  target - comma separated URL(s) to fetch (default: "http://localhost/")
+ requisites - if true also loads images and stylesheets referenced in the page as well, see wget --page-requisites
+              (default: "false")
+ cookies - if true saves and loads cookies to and from munin plugin state directory (default: "false")
```

Comment:
Please add a default assignment for the new `cookies` variable (just as it exists for `requisites` and `target` - around line 48). This clarifies the "interface" of the plugin in a single location.

And it would be nice, if you could describe the purpose (or *one* purpose) of the new `cookies` setting (i.e. re-use a single cookie-based session for all requests emitted by the plugin).